tests/service/ExprTests.fs

```diff
@@ -553,102 +649,2011 @@ let ``Test Declarations project1`` () =
+        "let tupleEx1 = (1,1) @ (9,16--9,21)";
+        "let tupleEx2 = (1,1,1) @ (10,16--10,25)";
+        "let tupleEx3 = (1,1,1,1) @ (11,16--11,29)";
+        "let localExample = let y: Microsoft.FSharp.Core.int = 1 in let z: Microsoft.FSharp.Core.int = 1 in (y,z) @ (14,7--14,8)";
+        "let localGenericFunctionExample(unitVar0) = let y: Microsoft.FSharp.Core.int = 1 in let compiledAsLocalGenericFunction: 'a -> 'a = FUN ... -> fun x -> x in (compiledAsLocalGenericFunction<Microsoft.FSharp.Core.int> y,compiledAsLocalGenericFunction<Microsoft.FSharp.Core.float> 1) @ (19,7--19,8)";
+        "let funcEx1(x) = x @ (23,23--23,24)";
+        "let genericFuncEx1(x) = x @ (24,29--24,30)";
+        "let topPair1b = M.patternInput@25 ().Item1 @ (25,4--25,26)";
+        "let topPair1a = M.patternInput@25 ().Item0 @ (25,4--25,26)";
+        "let testILCall1 = new Object() @ (27,18--27,27)";
+        "let testILCall2 = Console.WriteLine (\"176\") @ (28,18--28,49)";
+        "let recValNeverUsedAtRuntime = recValNeverUsedAtRuntime@31.Force<Microsoft.FSharp.Core.int>(()) @ (31,8--31,32)";
+        "let recFuncIgnoresFirstArg(g) (v) = v @ (32,33--32,34)";
+        "let testFun4(unitVar0) = let rec ... in recValNeverUsedAtRuntime @ (36,4--39,28)";
+        "type ClassWithImplicitConstructor";
+        "member .ctor(compiledAsArg) = (new Object(); (this.compiledAsArg <- compiledAsArg; (this.compiledAsField <- 1; let compiledAsLocal: Microsoft.FSharp.Core.int = 1 in let compiledAsLocal2: Microsoft.FSharp.Core.int = Operators.op_Addition<Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int> (compiledAsLocal,compiledAsLocal) in ()))) @ (41,5--41,33)";
+        "member .cctor(unitVar) = (compiledAsStaticField <- 1; let compiledAsStaticLocal: Microsoft.FSharp.Core.int = 1 in let compiledAsStaticLocal2: Microsoft.FSharp.Core.int = Operators.op_Addition<Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int> (compiledAsStaticLocal,compiledAsStaticLocal) in ()) @ (49,11--49,40)";
+        "member M1(__) (unitVar1) = Operators.op_Addition<Microsoft.FSharp.Core.int32,Microsoft.FSharp.Core.int32,Microsoft.FSharp.Core.int32> (Operators.op_Addition<Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int> (__.compiledAsField,__.compiledAsGenericInstanceMethod<Microsoft.FSharp.Core.int>(__.compiledAsField)),__.compiledAsArg) @ (55,21--55,102)";
+        "member M2(__) (unitVar1) = __.compiledAsInstanceMethod(()) @ (56,21--56,47)";
+        "member SM1(unitVar0) = Operators.op_Addition<Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int,Microsoft.FSharp.Core.int> (compiledAsStaticField,ClassWithImplicitConstructor.compiledAsGenericStaticMethod<Microsoft.FSharp.Core.int> (compiledAsStaticField)) @ (57,26--57,101)";
+        "member SM2(unitVar0) = ClassWithImplicitConstructor.compiledAsStaticMethod (()) @ (58,26--58,50)";
+#if NO_PROJECTCRACKER // proxy for COMPILER
```

Comment:
@ncave Do you recall why this was needed?  I don't think it's either NO_PROJECTCRACKER nor COMPILER. I'm hitting cases where this test is now failing and I'm not sure what I've done....

Reply:
@dsyme This just means the output of FSharp.LanguageService is slightly different than FCS, and accounts for that.

These two projects reference `FCS`:
`fcs/FSharp.Compiler.Service.Tests.netcore/FSharp.Compiler.Service.Tests.netcore.fsproj`
`fcs/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj`

but `vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj` references `FSharp.LanguageService`, so a symbol defined in `VisualFSharp.UnitTests.fsproj` is used to branch.

I don't know what is the error, perhaps the output is now the same? (if so, then there is no need to branch it anymore).

Reply:
> FSharp.LanguageService

I think you mean `FSharp.Compiler.Private` being used from `VisualFSharp.UnitTests.fsproj`

In https://github.com/Microsoft/visualfsharp/pull/4368/ I'm now always seeing the output in the ``NO_PROJECTCRACKER`` branch even from the two FCS test projects.  I don't particularly understand why though or what's changed.  Don't worry about it for now - let's see if CI passes